### ISSUE TITLE
Possible top bar bug

### ISSUE THREAD

**Issue by gordonpn** [cusec/2021#92]
When on mobile, the top bar is visible on the initial load and the page position is at the topmost. Once we scroll down and flick back up, the top bar is no longer visible when we are at the top again.

Is that the intended behaviour?

Please see the GIF animation below:

![demo](https://user-images.githubusercontent.com/4203037/102236226-3f819800-3ec1-11eb-9b3f-c252ffd8b23a.gif)

Also reproducible on Chrome mobile browser.

**Comment by sttic** [comment]
Good catch! I'm not sure what's happening there, it might be the 'bounce' from the flick is being interpreted as scrolling down (which hides the top bar)?

Also, fun fact (or not-so-fun fact), browsers on iOS are restricted to Apple's WebKit rendering/JS engines (see [Apple App Store guidelines](https://developer.apple.com/app-store/review/guidelines/#software-requirements) at point 2.5.6).

> Apps that browse the web must use the appropriate WebKit framework and WebKit Javascript.

So from my understanding, all browsers available on iOS are pretty much the same.
